test/literal/result.test.rb

```diff
@@ -71,23 +71,23 @@
 	class CardError < StandardError; end
 	class ConnectionError < StandardError; end
 
-	def fetch_something(result, &)
-		Literal::Result(Hash, &result).lift(CardError, ConnectionError, &)
+	def test_lift(&block)
```

Comment:
moved these nifty test helper methods so that I could `test_lift` in `lift!`, and put `.lift` below in its own block